config/routes.rb

```diff
@@ -34,6 +38,9 @@
   resources :attendees
   resources :events do
     resources :bookings, only: [:create]
+    #  member do
+    #   get "tickets"
+    # end
```

Comment:
hardcoded? do you mean i should stick to using nested resources instead of get? the nested resources is generating a url like tickets/1/events which is a bit confusing, instead of events/1/tickets, also when i was testing tickets in the events controller the action tickets was difficult to test, I was getting the error message no route matches tickets =>action in controller=> events